spec/support/spec/platform_data.rb

```diff
@@ -16,6 +16,55 @@ def platform_data(p, d)
       bin_dir: '/usr/bin',
       python_venv_version: '3.9',
     },
+    'Rocky-8' => {
```

Comment:
Can we copy RH 8 in code? Perhaps something like:
```ruby
p = case p
    when /Rocky-(\d+)/
     "RedHat-$1"
    else
      p
    end
```

And then do the same for the others